composer.json

```diff
@@ -70,6 +70,12 @@
         "update-dev": [
             "@composer update",
             "@cs-setup"
+        ],
+        "update-locales": [
+            "node_modules/ttag-cli/bin/ttag extract --o src/Locale/en_US/default-js.po --l en src/Template/Layout/js/app",
+            "node_modules/ttag-cli/bin/ttag extract --o src/Locale/it_IT/default-js.po --l it src/Template/Layout/js/app",
```

Comment:
should these commands be invoked by `bin/cake gettext update` instead??

Reply:
gettext suits only twig templates not ttag (.js) 
therefore there's 3 ways:
1. exec ttag node app from gettext update (very ugly, very quick)
2. modify gettext update so we can parse t`string` key words in .js files (almost useless ,very long)
3. use a quick node script that parse src/Locale and invoke ttag command (not clean, very quick)

i'd go with the third, even though web-tools should be able to do all by itself, it is also true that there's never gonna be a BE frontend app that doesn't depend on node

Reply:
I'd suggest using somehow env variables.
I.e.:
```
export $(cat .env | xargs) && ...
```

Reply:
Suggested solution:
* add a `ttag` extract step from `bin/cake gettext update` - do it only if `ttag` binary and `src/Template/Layout/js/app` folder exists - [why are we using the `Layout` folder anyway??]
* this step should update only the main `master.pot` - merge from master to available .po locales is already done by `gettext update` script - no need to do it twice. Locale .po files are the only i18n sources of truth.
* in the bundle creation copy locale .po files in the bundle - a more advanced solution (to implement later) copy only a subset of each .po files keeping only strings found in js files

Reply:
@stefanorosanelli if i remember @xho moved the js root under Layout sometime ago, nevertheless we already talked about doing changes to the folder structure